### PR TITLE
Add FAQ article vac_cert_mismatch

### PR DESCRIPTION
**Pull request by larswmh**
chrome_zFP8K47YQS
chrome_tEzuIFDFXQ

---
Internal Tracking ID: [EXPOSUREAPP-8069](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8069)

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -397,6 +397,17 @@
...
+                        "active": true,
+                        "textblock": [
+                            "Diese Fehlermeldungen treten in den Corona-Warn-App-Versionen 2.3 und 2.4 auf. Der genaue Wortlaut der Fehlermeldung lautet: „Die persönlichen Daten dieses Impfzertifikats stimmen nicht mit denen der bereits registrierten Zertifikate überein. Sie können in der App nur Zertifikate einer Person registrieren“",
+                            "Die App liest lediglich die im QR-Code des Impfzertifikates enthaltenen Daten aus und stellt diese leserlich in der App dar. Wenn bspw. in den beiden Impfzertifikaten Tippfehler oder unterschiedliche Schreibweisen genutzt wurden, erkennt die App diese nicht als zusammengehörig an. Typische Beispiele sind: zusätzliche Leerzeichen, Tippfehler im Namen oder aber in der Angabe eines Titels.",
+                            "Wurde das zweite Impfzertifikat korrekt ausgestellt, genügt es in der Regel, das erste Impfzertifikat in der Corona-Warn-App zu löschen und dann das zweite Impfzertifikat neu einzuscannen. Die App erkennt, dass auch damit ein vollständiger Impfschutz (nach 14 Tagen) erreicht wird.",
```
Comment (Ein-Tim) [inline src/data/faq_de.json:408]: Small wording change 
```suggestion
                            "Wurde das zweite Impfzertifikat korrekt ausgestellt, genügt es in der Regel, das erste Impfzertifikat aus der Corona-Warn-App zu löschen und dann das zweite Impfzertifikat neu einzuscannen. Die App erkennt, dass auch damit ein vollständiger Impfschutz (nach 14 Tagen) erreicht wird.",
```

**Review by Ein-Tim — COMMENTED** [review]
One very small comment (:

File: src/data/faq_de.json (hunk 1 above)
Comment (larswmh) [reply src/data/faq_de.json]: Hab ich so übernommen, vielen Dank

Comment (Ein-Tim) [reply src/data/faq_de.json]: Danke fürs übernehmen und Gerne doch!

**Comment by MikeMcC399** [comment]
@larswmh 
I suggest to use the same terms as used in the app:

EN 
Instead of "delete" use "remove"

DE
Instead of "löschen" use "entfernen"

Also, at least one user had a problem to find out how to remove a certificate from the app (see https://github.com/corona-warn-app/cwa-app-android/issues/3895) so maybe there could be an FAQ article to explain how to remove a vaccination certificate?

**Comment by larswmh** [comment]
@MikeMcC399 
sorry for the late answer. I will replace the terms in the next commit. Creating a FAQ article for the issue you mentioned seems valid,  ideally for both iOS and Android for the sake of completeness. Just so the work doesn't need to be done twice, did you maybe start creating such an article already? If not, I'd go ahead and create one based on your comment on the issue for Android and then my own step by step guide for iOS

**Comment by Ein-Tim** [comment]
@larswmh 

For iOS, you can use the instructions I posted here: https://github.com/corona-warn-app/cwa-app-android/issues/3895#issuecomment-895122236

**Comment by dsarkar** [comment]
@larswmh @MikeMcC399 @Ein-Tim Thanks!

**Comment by MikeMcC399** [comment]
@larswmh 
> Creating a FAQ article for the issue you mentioned seems valid, ideally for both iOS and Android for the sake of completeness. Just so the work doesn't need to be done twice, did you maybe start creating such an article already? If not, I'd go ahead and create one based on your comment on the issue for Android and then my own step by step guide for iOS

I don't have any other material prepared. I would not directly use my description in https://github.com/corona-warn-app/cwa-app-android/issues/3895#issuecomment-895132189. I think that needs to be reworked.

Thanks for your willingness to pick this up. In order to separate out the work, I created a new issue https://github.com/corona-warn-app/cwa-website/issues/1606 to discuss there further.

**Pull request merged**